po/tr.po

```diff
@@ -47,11 +47,11 @@ msgstr "Menü"
 
 #: src/MainWindow.vala:122
 msgid "Light content"
-msgstr "Açık İçerik"
+msgstr "Açık Tema"
```

Comment:
This seems to be a different translation… This should be translated as "Light content", not "Light Theme"; the idea is that it is controlling the web content, not the "theme" or style of the app itself.

Reply:
Sorry I thought it was related to the theme
I'm fixing